app/controllers/api/v2/user_columns_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module V2
+    class UserColumnsController < V2::BaseController
+      include Api::Version2
+
+      api :GET, "/user_columns/", N_("List of User columns for a user")
```

Comment:
user, capitalization should be consistent

Reply:
What about delete and update? Also I feel like` GET /user_columns` should return all the columns for each resource (would it ever make sense to get that info?) and `GET /user_columns/:resource` should return them for a resource